tests/aws/services/apigateway/test_apigateway_extended.py

```diff
@@ -63,15 +58,13 @@ def test_export_swagger_openapi(aws_client, snapshot, import_apigw, import_file,
     [TEST_IMPORT_PETSTORE_SWAGGER, TEST_IMPORT_PETS],
     ids=["TEST_IMPORT_PETSTORE_SWAGGER", "TEST_IMPORT_PETS"],
 )
-@markers.snapshot.skip_snapshot_verify(
-    paths=[
-        "$..body.servers..url",
-        "$..rootResourceId",
-    ]
-)
+@markers.snapshot.skip_snapshot_verify(paths=["$..body.servers..url"])
 def test_export_oas30_openapi(aws_client, snapshot, import_apigw, region_name, import_file):
     snapshot.add_transformer(
-        snapshot.transform.jsonpath("$.import-api.id", value_replacement="api-id")
+        [
+            snapshot.transform.jsonpath("$.import-api.id", value_replacement="api-id"),
+            snapshot.transform.key_value("rootResourceId"),
```

Comment:
It does, but in that test, we're not calling the `transformer_utility`, you would need to call `snapshot.add_transformer(snapshot.transform.apigateway_api())`. 

In that test case we were not adding it, I think one issue is `TransformerUtility.key_value("id")`, which can be pretty tricky sometimes because many, many resources have the `id` field and it starts doing weird thing.